packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -255,12 +255,18 @@ export class AuthCodeRedirectHandler implements IRedirectHandler {
         secure: false,
       }
     );
-
-    await setupResourceServerSession(
-      tokens.webId,
-      authFetch,
-      this.storageUtility
-    );
+    // TODO: This is a temporary workaround. When deprecating the cookie-based auth,
+    // this should be all cleared.
+    const essWorkaroundDisabled =
+      window.localStorage.getItem("tmp-resource-server-session-enabled") ===
+      "false";
+    if (!essWorkaroundDisabled) {
```

Comment:
Ha, so if the disabled value that represents whether the enabled LocalStorage option is equal to the string "false" is not true, this block holds? :P 

(It's readable just fine, I was just amused by the quadruple negative :) )